octobot_trading/channels/ticker.py

```diff
@@ -29,7 +29,7 @@ async def perform(self, symbol, ticker):
             if self.channel.get_filtered_consumers(symbol=CHANNEL_WILDCARD) or \
                     self.channel.get_filtered_consumers(symbol=symbol):  #
                 if ticker:  # and price_ticker_is_initialized
-                    self.channel.exchange_manager.get_symbol_data(symbol).handle_ticker_update(ticker)
```

Comment:
Ticker wont be updated anymore in SymbolData

Reply:
that is a mistake, I'm fixing it